hapi/hedera-protobufs/services/response_code.proto

```diff
@@ -1593,4 +1593,26 @@ enum ResponseCodeEnum {
      * airdrop and whether the sender can fulfill the offer.
      */
     INVALID_TOKEN_IN_PENDING_AIRDROP = 369;
+
+    /**
+    * The provided fee exempt key list size exceeded the limit.
+    */
+    MAX_ENTRIES_FOR_FEKL_EXCEEDED = 370;
```

Comment:
We can use `FEE_EXEMPT_KEY_LIST` instead of FEKL here ? Just a thought for clarity. I see HIP mentions FEKL also, but since its returned to users FEKL might not be clear.

Reply:
Fixed, thanks for the suggestion.